.github/documentation_2020-04-11.json

```diff
@@ -0,0 +1 @@
974e8dae3c1089abeb3899","authentication":{},"body":{},"created":1586650318401,"description":"","headers":[],"isPrivate":false,"metaSortKey":-1586650318401,"method":"GET","modified":1586650360723,"name":"Get Community Content","parameters":[],"parentId":"fld_ab3ec0c92a744cd9a40b2c3e53d6afa3","settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingFollowRedirects":"global","settingRebuildPath":true,"settingSendCookies":true,"settingStoreCookies":true,"url":"{{ baseUrl  }}community","_type":"request"},{"_id":"fld_ab3ec0c92a744cd9a40b2c3e53d6afa3","created":1586650105247,"description":"","environment":{},"environmentPropertyOrder":null,"metaSortKey":-1586650105247,"modified":1586650105247,"name":"Community","parentId":"wrk_24535709b14f4b878975217c89dc606f","_type":"request_group"},{"_id":"req_fa06f65b2feb42419e357c873ee49e52","authentication":{},"body":{},"created":1586650338239,"description":"","headers":[],"isPrivate":false,"metaSortKey":-1586650338239,"method":"GET","modified":1586650357708,"name":"Get Youtube Content","parameters":[],"parentId":"fld_dbc8b1056db442c9abfaa911d1f3fb05","settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingFollowRedirects":"global","settingRebuildPath":true,"settingSendCookies":true,"settingStoreCookies":true,"url":"{{ baseUrl  }}youtube","_type":"request"},{"_id":"fld_dbc8b1056db442c9abfaa911d1f3fb05","created":1586650099666,"description":"","environment":{},"environmentPropertyOrder":null,"metaSortKey":-1586650099666,"modified":1586650099666,"name":"Youtube","parentId":"wrk_24535709b14f4b878975217c89dc606f","_type":"request_group"},{"_id":"req_cf9348ede7824bcab4ff7a6f503adbc0","authentication":{},"body":{},"created":1586650353654,"description":"","headers":[],"isPrivate":false,"metaSortKey":-1586650353654,"method":"GET","modified":1586650378017,"name":"Get Twitch Content","parameters":[],"parentId":"fld_3957d7484a1242e08a1d5a25a93b4be1","settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingFollowRedirects":"global","settingRebuildPath":true,"settingSendCookies":true,"settingStoreCookies":true,"url":"{{ baseUrl  }}twitch","_type":"request"},{"_id":"fld_3957d7484a1242e08a1d5a25a93b4be1","created":1586650094631,"description":"","environment":{},"environmentPropertyOrder":null,"metaSortKey":-1586650094631,"modified":1586650094631,"name":"Twitch","parentId":"wrk_24535709b14f4b878975217c89dc606f","_type":"request_group"},{"_id":"env_260ba5ef13fe1275062808671a2159189a9447be","color":null,"created":1586650084666,"data":{"baseUrl":"https://quarentena.tech/api/"},"dataPropertyOrder":{"&":["baseUrl"]},"isPrivate":false,"metaSortKey":1586650084666,"modified":1586650210087,"name":"Base Environment","parentId":"wrk_24535709b14f4b878975217c89dc606f","_type":"environment"},{"_id":"jar_260ba5ef13fe1275062808671a2159189a9447be","cookies":[],"created":1586650084682,"modified":1586650084682,"name":"Default Jar","parentId":"wrk_24535709b14f4b878975217c89dc606f","_type":"cookie_jar"}]}
```

Comment:
O que é esse arquivo?

Reply:
é o arquivo de documentação do insomnia. Quando a pessoa clicar no botão "Run in Insomnia" ele vai ser baixado por ela e aberto pelo insomnia já com toda configuração dos endpoints da api, caso existam mais endpoints depois, quando adicionados no insomnia é só exportar e salvar o arquivo na pasta .github e ai terá a documentação atualizada no click to run.